prime-router/src/test/kotlin/history/azure/SubmissionFunctionIntegrationTests.kt

```diff
@@ -153,7 +307,7 @@ class SubmissionFunctionIntegrationTests {
         val historyNode = JacksonMapperUtilities.defaultMapper.readTree(history.body.toString())
         assertThat(
             historyNode.get("overallStatus").asText()
-        ).isEqualTo(DetailedSubmissionHistory.Status.PARTIALLY_DELIVERED.toString())
+        ).isEqualTo(DetailedSubmissionHistory.Status.PARTIALLY_DELIVERED.toString()) // TODO: should be RECEIVED
```

Comment:
why the TODO?

Reply:
Related to the todo in the function header (#16054)

Reply:
Can you please include the ticket number in the TODO?